change/@azure-msal-browser-6b150027-dba7-407a-a266-afa6dc7fe44f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
This seems more like a bug fix than a new feature (esp given public API isnt changing)

Reply:
Right, I confused myself since we added a required attribute to a "public" (but not public facing) API.